src/Microsoft.VisualStudio.SlowCheetah.VS/Resources/Guids.cs

```diff
@@ -18,7 +18,7 @@ public static class Guids
         /// <summary>
         /// Guid string of a Web Application project
         /// </summary>
-        public const string GuidWebApplicationString = "{349c5851-65df-11da-9384-00065b846f21}";
+        public static readonly Guid GuidWebApplicationString = new Guid("349c5851-65df-11da-9384-00065b846f21");
```

Comment:
This only changes the value. but the name of the variable and the documentation is unchanged. It's probably better to create a new variable for the GUID, as is done with `GuidSlowCheetahCmdSet`